articles/azure-vmware/migrate-sql-server-standalone-cluster.md

```diff
@@ -72,11 +78,13 @@ This table indicates the estimated downtime for each Microsoft SQL Server topolo
    - Select **Go** to start the migration.
 1. After the migration has completed, access the virtual machine using VMware Remote Console in the vSphere Client.
    - Verify the network configuration and check connectivity both with on-premises and Azure VMware Solution resources.
-   - Using Microsoft SQL Server Management Studio verify you can access the database.  
+1. Verify your SQL Server and databases are up and accessible. For example, using SQL Server Management Studio verify you can access the database.  
```

Comment:
```suggestion
1. Verify your SQL Server and databases are up and accessible. For example, using SQL Server Management Studio verify you can access the database.  
```
Making this a numbered step leaves a single bullet in the previous step, which is not a best practice. Can you fix?